nemo_curator/modules/semantic_dedup.py

```diff
@@ -171,6 +177,9 @@ def __init__(
         self.model = EmbeddingCrossFitModel(self.embeddings_config)
         self.write_embeddings_to_disk = write_embeddings_to_disk
         self.write_to_filename = write_to_filename
+        if input_file_type == "json":
+            input_file_type = "jsonl"
+        self.output_file_type = input_file_type.lower()
```

Comment:
We should not force the same `input_file_type` and `output_file_type`.  We should just stick with `parquet` internally to help performance.